scripts/set-page-title.py

```diff
@@ -0,0 +1,197 @@
+        # return empty status to indicate that no changes were made
+        return ""
+
+    if dry_run:
+        status = "title would be updated"
+    else:
+        status = "title updated"
+
+    status = f"\x1b[34m{status}\x1b[0m"
+
+    if not dry_run:
+        lines[0] = new_line
+        with path.open("w", encoding="utf-8") as f:
+            f.writelines(lines)
+
+    return status
+
+
+def get_title(path: Path) -> str:
+    with path.open(encoding="utf-8") as f:
+        first_line = f.readline().strip()
+
+    return first_line.split("#", 1)[-1].strip()
+
+
+def sync(
+    root: Path, pages_dirs: list[str], command: str, title: str, dry_run=False
+) -> list[str]:
+    paths = []
+    for page_dir in pages_dirs:
+        path = root / page_dir / command
+        if path.exists():
+            rel_path = "/".join(path.parts[-3:])
+            status = set_title(path, title, dry_run)
+            if status != "":
+                paths.append(path)
+                print(f"\x1b[32m{rel_path} {status}\x1b[0m")
+    return paths
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description="Sets the title for all translations of a page"
+    )
+    parser.add_argument(
+        "-p",
+        "--page",
+        type=str,
+        required=False,
+        default="",
+        help='page name in the format "platform/command.md"',
+    )
+    parser.add_argument(
+        "-s",
+        "--stage",
+        action="store_true",
+        default=False,
+        help="stage modified pages (requires `git` to be on $PATH and TLDR_ROOT to be a Git repository)",
+    )
+    parser.add_argument(
+        "-S",
+        "--sync",
+        action="store_true",
+        default=False,
+        help="synchronize each translation's title (if exists) with that of English page",
+    )
+    parser.add_argument(
+        "-n",
+        "--dry-run",
+        action="store_true",
+        default=False,
+        help="show what changes would be made without actually modifying the pages",
+    )
+    parser.add_argument("title", type=str, nargs="?", default="")
+    args = parser.parse_args()
+
+    root = get_tldr_root()
+    pages_dirs = [d for d in root.iterdir() if d.name.startswith("pages")]
+
+    target_paths = []
+
+    # Use '--page' option
+    if args.page != "":
+        if not args.page.lower().endswith(".md"):
+            args.page = f"{args.page}.md"
+        arg_platform, arg_page = args.page.split("/")
+
+        for pages_dir in pages_dirs:
+            page_path = pages_dir / arg_platform / arg_page
+            if not page_path.exists():
+                continue
+            target_paths.append(page_path)
+
+        target_paths.sort()
+
+        for path in target_paths:
+            rel_path = "/".join(path.parts[-3:])
+            status = set_title(path, args.title)
+            if status != "":
+                print(f"\x1b[32m{rel_path} {status}\x1b[0m")
+
+    # Use '--sync' option
+    elif args.sync:
```

Comment:
I just noticed this problem now (it was already present on `set-alias-page.py` and `set-more-info-link.py`), but even though "--sync" and "--page" are **options** (they are optional, and not required), they are, in reality, mutually exclusive.

These "required and mutually exclusive options" can confuse anyone that uses these scripts for the first time. Particularly because it's fairly intuitive to think they can be used together. For example: in set-page-title.py, one can think, without paying much attention to the help message or documentation - "--page" alone just sets the title in the English page, while "--sync" synchronizes this change across all translations, but this is not correct.

I understand "required and mutually exclusive options" is somewhat common across the CLI commands, but we can always improve :) A better solution is to use subcommands, they communicate the idea that two functionalities are mutually exclusive in a much clearer way.

I think this is not that hard to implement this, Python has what they call "subparsers" that allows the implementation of sub-commands. They are well explained in their official documentation: https://docs.python.org/3/library/argparse.html#sub-commands